### DISCUSSION TITLE
github page

### DISCUSSION THREAD

**Discussion by SegoviaId** [community/community#28873] [Actions]
https://user-images.githubusercontent.com/110736156/183616566-0ae7c3b4-ccf0-4a52-bb57-a815c12fb55d.mp4

![Capture git p](https://user-images.githubusercontent.com/110736156/183617238-f94263e7-ce28-44f0-8182-2c3ec344f1a5.png)

Hello,

I can't create the link (github page) mon repositorie . 

Thanks.

**Comment by airtower-luna** [comment]
Your Pages site is enabled, but the build fails (check the Actions tab). The latest build failed because of the two invalid submodules in your repository: https://github.com/SegoviaId/P3/runs/7743010626?check_suite_focus=true

You'll need to either fix (so the build can download them) or remove the submodules.

**Reply by SegoviaId** [reply]
Thank you!!
How are they repaired?
I'm starting to dev

**Reply by airtower-luna** [reply]
I see you already removed one of the submodules, that's a good start. For the `images` one, the question is: From where do you want to get the images? Should that be a local directory, or do you have another repository that contains the images?

**Reply by SegoviaId** [reply]
In theory I have uploaded the complete file with the images. The sass file is identical and they have no problems.
![Capture d’écran (247)](https://user-images.githubusercontent.com/110736156/183701023-85fe283c-2492-42ac-8913-5dd527068894.png)
![Capture d’écran (248)](https://user-images.githubusercontent.com/110736156/183701027-ec742c3d-fc51-48f0-95e1-cef9b7f6bb6f.png)

**Reply by SegoviaId** [reply]
I don't know why it comes out as a submodule.

Thank you!

**Reply by airtower-luna** [reply]
Maybe you accidentally turned `images/` into a (nested) Git repository instead of adding the files inside to the one for your site? In that case there'll be a (hidden) `images/.git/` directory that you'll need to remove, and then make sure the submodule config is gone, too.

After that add and commit the image files.
